### ISSUE TITLE
ol.style.Icon appears to still not work in Safari in v6.0.0-beta.11 

### ISSUE THREAD

**Issue by CameronWatt98** [openlayers/openlayers#9813]
**Describe the bug**
When using Safari on an iPhone 6 running IOS12.3.1 my svg icons do not render. However they render on a windows PC and an Android Tablet/Phone. I have updated my angular app to use v6.0.0-beta.11 from 5.3.3 as I was told that a a fix had been committed here:
https://github.com/openlayers/openlayers/pull/9658/commits/002dc344557defb4549c27abcf27d8a127eccbb3
I was told this fix was available in V6.0.0.-beta.11, however even with an updated application my Icons do not render. 

**To Reproduce**
This is the code I use to create my icons
 var iconStyle = new Style({
                image: new Icon(({
                  color: mycol,
                  opacity: myopacity,
                  scale: 1,
                  src: mypicture.svg'
                }))
              });
**Expected behavior**
I would expect the icons to render when using any browser on all platforms.

**Comment by CameronWatt98** [comment]
It may also be beneficial to know that I create features that add a tail on the Icons and they also do not render on Safari on IOS.  

        var iconFeature2 = new Feature({
          geometry: new LineString(track),
              name: Name,
               type: VesselType
        });
        var iconStyle2 = new Style({
          stroke: new Stroke({
            color: col2,
            width: 1
          }),
        });
        iconFeature2.setStyle(iconStyle2);

**Comment by ahocevar** [comment]
To give this issue a better chance to get fixed, you may want to provide either a fully working example (e.g. through codesandbox.io), or at least the svg file you are using.

**Comment by CameronWatt98** [comment]
I had attached a zip archive containing the .svg file.
[Survey.zip](https://github.com/openlayers/openlayers/files/3452559/Survey.zip)

**Comment by CameronWatt98** [comment]
I get the map data from an ArcGis Map Server then use that ImageArcGisRest to make a base image layer. I then have a vector layer on top that contains many icons and their trails. It looks like the image below:
![Map](https://user-images.githubusercontent.com/31926426/62225009-809e0880-b3af-11e9-9783-d8f216aecfa0.JPG)

However on Safari it shows just the base layer like this:

![Map blank](https://user-images.githubusercontent.com/31926426/62225098-b511c480-b3af-11e9-8d16-121e9c5bf129.JPG)

**Comment by CameronWatt98** [comment]
I have found that it also doesn't display if I use .png or .jpg images either. I also used LambaTest a cross-browser testing tool and found that it also doesn't work on Safari on MacOSX. However, I did notice a new error crop up when testing on Safari (image below). It wasn't clear if it had anything to do with the issue but I thought I would share it. (Apologies for the bad images, I couldn't copy the code)
![InkedCapture2_LI](https://user-images.githubusercontent.com/31926426/62556785-8727f680-b86d-11e9-9aee-a3764604936a.jpg)
![InkedCapture1_LI](https://user-images.githubusercontent.com/31926426/62556763-8000e880-b86d-11e9-9489-22e21563fa35.jpg)

This is the actual error message:

![Capture3](https://user-images.githubusercontent.com/31926426/62556794-8c854100-b86d-11e9-8b28-18bb0d4afdcd.JPG)
